packages/dev/core/src/Meshes/geometry.ts

```diff
@@ -793,7 +793,10 @@ export class Geometry implements IGetSetVerticesData {
                 if (!this._extend) {
                     this._updateExtend();
                 }
-                mesh.buildBoundingInfo(this._extend.minimum, this._extend.maximum);
+                // If the data have not been uploaded by Babylon (ex: Compute Shaders and Storage Buffers), _extend may be undefined even after calling _updateExtend()
+                if (this._extend) {

```

Comment:
Instead of doing this check, is there some reasonable default value we can initialize `_extend` to? Like either 0,0,0 for min/max (no bounds) or Number.MIN_VALUE/Number.MAX_VALUE (infinite bounds)? It would be great to enable the TS `strictPropertyInitialization` compiler flag to avoid these bugs, but if/when we do that, we'll need to either have reasonable defaults for these values, or make breaking changes to make them Nullable if they really can't have any reasonable defaults.

Reply:
Can it be initialized to this value on line 68, rather than being initialized lazily?